docs: add advanced flagd links (#492)
open-feature/open-feature-operator: docs/getting_started.md

```sh
{"value":"BAR","reason":"STATIC","variant":"bar"}
```

> For more advanced flagd syntax, such as targeting rules, check these examples in [JSON format](https://github.com/open-feature/flagd/blob/main/samples/example_flags.flagd.json) or [YAML format](https://github.com/open-feature/flagd/blob/main/samples/example_flags.flagd.yaml).

Reviewer: ```suggestion
For more advanced flagd syntax, such as targeting rules, check the [flag configuration documentation](https://github.com/open-feature/flagd/blob/main/docs/configuration/flag_configuration.md).
```
